lib/recurly/traits/xml_doc.php

```diff
@@ -83,6 +83,7 @@ protected function getChangedAttributes($nested = false)
     $writableAttributes = $this->getWriteableAttributes();
     $requiredAttributes = $this->getRequiredAttributes();
 
+    print_r($this->_unsavedKeys);
```

Comment:
Intentionally leaving this?

Reply:
Nope, forgot to save that after removing it